kubejs/server_scripts/farmerscompat.js

```diff
@@ -0,0 +1,53 @@
+onEvent("recipes", (event) => {
+    mixingRecipeGen(event);
+})
+
+var FULL_BUCKET_AMMOUNT = 81000;
+var INGOT_FLUID_AMMOUNT = 9000;
+
+function mixingRecipeGen(event) {
+    event.forEachRecipe({type:"farmersdelight:cooking"}, recipe => {
+        //Gets item name without namespace (e.g. "cooked_rice")
+        var outputItem = recipe.getOriginalRecipeResult().getItem().getId().substring(recipe.getOriginalRecipeResult().getItem().getId().indexOf(":")+1); 

```

Comment:
For cooked rice it would be "farmersdelight:cooked_rice"

I did that whole thing so I could just add "_fluid" to the end of that variable to refer to the fluid variant of that item that I made in startup.js. I could assign recipe.getOriginalRecipeResult().getItem().getI() to a variable earlier to make it less messy.

Reply:
what about `recipe.getOriginalRecipeResult().split(":")[1]`